store/tikv/latch/latch.go

```diff
@@ -69,7 +72,7 @@ func NewLock(startTS uint64, requiredSlots []int) Lock {
 type Latches struct {
 	slots []Latch
 	// The waiting queue for each slot(slotID => slice of startTS).
-	waitingQueues map[int][]uint64
+	waitingQueues map[int][]*Lock
```

Comment:
This change will conflict with https://github.com/pingcap/tidb/pull/6298
Please only fix the test.